admin/plugins/civicrm/civicrm.php

```diff
@@ -7,6 +7,7 @@
 // No direct access
 defined('_JEXEC') or die;
 
+// TODO: remove the below once JPlugin no longer referenced for J4+
 jimport('joomla.plugin.plugin');
```

Comment:
We don't have similar `jimport()` statements for the other CiviCRM plugin files (to bring in the `JPlugin` class), which makes me think that we can just remove this line entirely.

It doesn't make any difference if I remove it on our J5 instance.

I don't have a J3 instance to test this on though - does anyone know more about this?